easybuild/easyblocks/t/tensorflow.py

```diff
@@ -194,6 +221,8 @@ def configure_step(self):
             env.setvar(key, val)
 
         # patch configure.py (called by configure script) to avoid that Bazel abuses $HOME/.cache/bazel
+        # Should perhaps set --install_base  and --output_user_root too
+        # bazel still uses $HOME/.cache/bazel for some things.
```

Comment:
@akesandgren open an issue for this, no point in keeping track of TODOs in comments (especially without mentioning `TODO` or `FIXME` ;-))